src/measure_theory/integral/circle_integral.lean

```diff
@@ -0,0 +1,513 @@
+/-
+Copyright (c) 2021 Yury Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury Kudryashov
+-/
+import measure_theory.integral.interval_integral
+import analysis.normed_space.pointwise
+import analysis.special_functions.non_integrable
+import analysis.analytic.basic
+
+/-!
+# Integral over a circle in `ℂ`
+
+In this file we define `∮ z in C(c, R), f z` to be the integral $\oint_{|z-c|=|R|} f(z)\,dz$ and
+prove some properties of this integral.
+
+## Main definitions
+
+* `circle_map c R`: the exponential map $θ ↦ c + R e^{θi}$;
+
+* `circle_integrable f c R`: a function `f : ℂ → E` is integrable on the circle with center `c` and
```

Comment:
Could you also include some discussion of what `E` is? Many people will come here expecting `E =  ℂ` everywhere.